tests/test_download_utils.py

```diff
@@ -42,63 +43,63 @@ def test_downloader(mocker, mock_path):
     )
 
     zip_remote = download_utils.RemoteFileMetadata(
-        filename="remote.zip", url="a", checksum="1234"
+        filename="remote.zip", url="a", checksum=("1234")
```

Comment:
Sorry but I'm missing why you need the `()` here, could you explain?

Reply:
I do not remember doing this 😯 I could think that it was done by mistake, but it has been done consistently across all the code (see lines 49, 53, 164, 167, etc) Maybe a black/tox thing?